includes/uploader.js

```diff
@@ -104,11 +112,17 @@ function check_file_content(isValidFile) {
                 columnNames = content.split('\n')[0].split(',');
 
                 // Log column names
-                console.log('Column Names:', columnNames);
+                //console.log('Column Names:', columnNames);
 
                 // Extract and log each row
                 rowValues = content.split('\n').slice(1).map(row => row.split(','));
-                console.log('Row Values:', rowValues);
+                //console.log('Row Values:', rowValues);
+
+                // Rows and columns are properly separated, check columns names
+                isValidContent = true;
+                if (isValidContent) {
+                    check_columns_names(isValidContent);

```

Comment:
Shall we move this check to the start of the function, so after reading the `columnNames` variable? That means we can 'error' out and halt the process as soon as the header is incorrect, without having to wait for the rest of the rows to be processed.